sympy/utilities/pytest.py

```diff
@@ -90,6 +91,43 @@ def raises(expectedException, code=None):
             raise TypeError(
                 'raises() expects a callable for the 2nd argument.')
```

Comment:
How about adding an argument to existing `raises`?

Reply:
That can be done, with default `err_msg=""` (empty string).

Reply:
@gxyd Now I think, in terms of `pytest` compatibility, `raises` API should match theirs.